lib/graphql/static_validation/rules/unique_directives_per_location.rb

```diff
@@ -0,0 +1,38 @@
+module GraphQL
+  module StaticValidation
+    class UniqueDirectivesPerLocation
+      include GraphQL::StaticValidation::Message::MessageHelper
+
+      def validate(context)
+        nodes_with_directives = GraphQL::Language::Nodes.constants
+          .map{|c| GraphQL::Language::Nodes.const_get(c)}
+          .select{|c| c.is_a?(Class) && c.instance_methods.include?(:directives)}
```

Comment:
Seems fine to me, but does it need to happen for _each_ query, or could we move it to a constant, and use that constant in `#validate`?